src/lib.rs

```diff
@@ -1,9 +1,9 @@
 #![deny(
-    missing_docs,
```

Comment:
I'm slightly confused as to why reordering the parameters fixes the nightly build. Do you have an explanation or reference?